loadouts/includes/private-variables.sqf

```diff
@@ -36,30 +36,37 @@ private [
 
 	"_factionSkill", "_factionUnits", "_factionVehicles", "_factionObjects", "_factionDACCamps",
 
-	"_commonRifle", "_countRifle", "_countRifleLOW", "_countTracerRIFLE", "_countRifleCARGO",
-	"_commonRifleGL", "_count40mm", "_count40mmCARGO",
+	"_randomLIST","_rifleLIST",
+	"_commonRIFLE", "_countRifle", "_countRifleLOW", "_countTracerRIFLE", "_countRifleCARGO","_specRIFLE",
+	"_commonRIFLEGL", "_count40mm", "_count40mmCARGO",
 	"_commonPistol", "_countPISTOL", "_countPistolCARGO",
 	"_commonAR", "_countAR", "_countArCARGO",
 	"_commonMG", "_countMG", "_countMgCARGO", "_countTracerMG",
 	"_commonMarksman",
 	"_commonSniper", "_countSniper", "_countSniperCARGO",
 	"_commonAT", "_countAT", "_countATCARGO",
 	"_specAT", "_countSpecATCARGO", "_isSpecATDisposable",
-	"_commonSMG",
+	"_commonSMG","_commonGL",
 	"_weaponsAA", "_countWeaponsAA", "_countWeaponsAACARGO",
 	"_weaponsAT", "_countWeaponsAT", "_countWeaponsATCARGO",
 
-	"_commonRCO", "_commonCCO", "_commonMAGNIFIED",
+	"_commonRCO", "_specCCO", "_commonCCO","_commonEOT", "_commonMAGNIFIED","_specRCO","_atRCO",
 	"_commonSUPPRESSOR", "_commonPISTOLSUPPRESSOR",
+	"_commonLAZER","_commonGRIP",
+
+	"_rifleSCOPE","_mgSCOPE","_specSCOPE",
+
+	"_binoc","_rangefinder","_laserdesignator",
 
 	"_countGRENADES", "_countGrenadesCARGO",
 
 	"_countHE",
 
 	"_commonHEAD", "_leaderHEAD", "_officerHEAD", "_medicHEAD", "_crewmanHEAD", "_pilotHEAD", "_helicrewHEAD", "_helipilotHEAD", "_sniperHEAD", "_demoHEAD", "_reconHEAD",
 	"_commonUNIFORM", "_officerUNIFORM", "_pilotUNIFORM", "_sniperUNIFORM", "_marksmanUNIFORM", "_helicrewUNIFORM", "_crewUNIFORM", "_mgUNIFORM", "_medicUNIFORM", "_demoUNIFORM", "_reconUNIFORM",
-	"_commonVEST", "_officerVEST", "_ftlVEST", "_slVEST","_mgVEST", "_grenadierVEST", "_medicVEST", "_demoVEST", "_marksmanVEST", "_reconVEST",
-	"_commonBACKPACK", "_bigBACKPACK",
+	"_commonVEST", "_officerVEST", "_ftlVEST", "_slVEST","_mgVEST", "_grenadierVEST", "_atVEST", "_medicVEST", "_demoVEST", "_marksmanVEST", "_reconVEST","_pilotVEST",
+	"_commonBACKPACK", "_bigBACKPACK","_medicalBACKPACK","_atBACKPACK","_reconBACKPACK",
```

Comment:
`_medicalBACKPACK` isn't used in any structures.
Rename `_atBACKPACK` to `_specATBACKPACK`. Also, only used in low and mid tier structures, but is set in a few high tier factions.